docs/en/latest/plugins/ai-prompt-decorator.md

```diff
@@ -0,0 +1,112 @@
+---
+title: ai-prompt-decorator
+keywords:
+  - Apache APISIX
+  - API Gateway
+  - Plugin
+  - ai-prompt-decorator
+description: This document contains information about the Apache APISIX ai-prompt-decorator Plugin.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `ai-prompt-decorator` plugin simplifies access to AI providers and models by appending or prepending
+prompts into the request.
+
+## Plugin Attributes
+
+| **Field**         | **Type** | **Description**                                     | **Required**    |
+| ----------------- | -------- | --------------------------------------------------- | --------------- |
+| `prepend`         | Array    | An array of prompt objects to be prepended          | Conditionally\* |
+| `prepend.role`    | String   | Role of the message (`system`, `user`, `assistant`) | Yes             |
+| `prepend.content` | String   | Content of the message. Minimum length: 1           | Yes             |
+| `append`          | Array    | An array of prompt objects to be appended           | Conditionally\* |
+| `append.role`     | String   | Role of the message (`system`, `user`, `assistant`) | Yes             |
+| `append.content`  | String   | Content of the message. Minimum length: 1           | Yes             |
```

Comment:
Move the column to the front?

<img width="818" alt="image" src="https://github.com/user-attachments/assets/b62f1031-3b11-4628-819d-735e11334234">